spec/features/users/regenerate_personal_key_spec.rb

```diff
@@ -119,10 +119,9 @@
       click_acknowledge_personal_key
 
       expect_confirmation_modal_to_appear_with_first_code_field_in_focus
-
-      press_tab
-
-      expect_continue_button_to_be_in_focus
+      within('.sm-hide') do
+        expect(page).to have_button t('forms.buttons.continue')
+      end
```

Comment:
For some reason, after pressing tab, Chrome didn't see that the continue button was in focus. What we want to test here is that on a mobile device, the continue button appears first, and the way this was implemented was to duplicate the button, once with a `.sm-hide` and once with a `.sm-show`: https://github.com/18F/identity-idp/commit/c9b499f9c9baa857106f0946a92ae2d13da3b212

The `.sm-hide` class only shows up at a certain screen size, so this is good enough for me.